sentry_sdk/tracing.py

```diff
@@ -135,7 +135,7 @@ def __init__(
         sampled=None,  # type: Optional[bool]
         op=None,  # type: Optional[str]
         description=None,  # type: Optional[str]
-        hub=None,  # type: Optional[sentry_sdk.Hub]
+        hub=None,  # type: Optional[Union[sentry_sdk.Hub, sentry_sdk.Scope]]
```

Comment:
Should we additionally accept `scope` as optional arg here so that we can deprecate and throw `hub` away at some point, but still support using a custom `scope` for a span?